tests/parameters.test.ts

```diff
@@ -1,59 +1,112 @@
 import { transformParametersArray } from "../src/transform/parameters";
 
-describe("transformParametersArray()", () => {
-  it("basic", () => {
-    expect(
-      transformParametersArray([
-        {
-          description: "Specifies which fields in the response should be expanded.",
-          in: "query",
-          name: "expand",
-          required: false,
-          schema: {
-            items: {
-              type: "string",
+describe.only("transformParametersArray()", () => {
+  describe("v2", () => {
+    it("basic", () => {
+      expect(
+        transformParametersArray(
+          [
+            {
+              description: "Specifies which fields in the response should be expanded.",
+              in: "query",
+              name: "expand",
+              required: false,
+              items: {
+                type: "string",
+              },
+              type: "array",
             },
-            type: "array",
-          },
-        },
-        {
-          in: "path",
-          name: "three_d_secure",
-          required: true,
-          schema: {
-            type: "string",
-          },
-        },
-      ]).trim()
-    ).toBe(
-      `query: {
+            { in: "path", name: "three_d_secure", required: true, type: "string" },
```

Comment:
Suggestion: with our new knowledge, let’s try adding a `body` param here, like: `{ in: "body", schema: { type: "string" } }`. Then we can test if it generated correctly.